Getting_Ready_to_Chess.cpp

```diff
@@ -199,7 +21,7 @@ int main()
     while (!koordinates.eof())
     {
```

Comment:
please put all code for reading from file to a separate function, e. g. ReadFiguresFromFile()